getting-started/README.md

```diff
@@ -15,11 +15,11 @@ To use the app, you need:
 
 ## Usage
 
-Read the [Getting Started](https://www.pubnub.com/docs/chat/components/ios/get-started-ios) guide to learn how to use the app and better understand the logic behind it.
+Read the [Getting Started](https://www.pubnub.com/docs/chat/components/ios/get-started) guide to learn how to use the app and better understand the logic behind it.
```

Comment:
the same as above, previous one was correct

Reply:
Fixed.